pkg/platform/common/vmlayer/lb.go

```diff
@@ -49,6 +49,7 @@ var udevRulesFile = "/etc/udev/rules.d/70-persistent-net.rules"
 var sharedRootLBPortLock sync.Mutex
 
 var RootLBPorts = []dme.AppPort{}
+var NoAccessKey string
```

Comment:
Not used anywhere.

Reply:
It's still used in calls to GetVMSpecForRootLB in common/vmlayer/appinst.go and clusterinst.go. I do see it's not actually used in the func, so now I changed the check in GetVMSpecForRootLB to compare accessKey to `NoAccessKey` instead of `""`.